src/promo/ThinBanner.test.jsx

```diff
@@ -0,0 +1,71 @@
+import Button from '../Button'
+import ArrowRightIcon from '@material-ui/icons/KeyboardArrowRight'
+import ThinBanner from './ThinBanner'
+import React from 'react'
+import { mount, shallow } from 'enzyme'
```

Comment:
```suggestion
import { mount } from 'enzyme'
```

Since all `shallow` wrappers are being `.dive`d upon, you could simply `mount` them and replace all `wrapper.dive()` with `wrapper`.